tests/integration/userTests.py

```diff
@@ -209,9 +209,9 @@ def test_list_submissions(self):
         response = self.app.get("/v1/list_submissions/", headers={"x-session-id": self.session_id})
         self.check_response(response, StatusCode.OK)
         self.assertIn("submissions", response.json)
-        self.assertEqual(len(response.json["submissions"]), 6)
+        self.assertEqual(len(response.json["submissions"]), 5)
```

Comment:
If I understand correctly, this is `5` due to the new page size; mind adding a comment to explain either way where `5` comes from?